config/initializers/bugsnag.rb

```diff
@@ -1,4 +1,6 @@
-Bugsnag.configure do |config|
-  config.api_key = ENV['BUGSNAG_API_KEY']
-  config.notify_release_stages = %w(production staging)
+# Bugsnag will notify for any environment that the gem is installed (see Gemfile)
```

Comment:
Interesting approach. So if we want to test something in development then we need to comment out the group limitation. That's easier than adding development in the initializer. :heavy_check_mark: 

It looks like Bugsnag would run in test environment now though. But since the tests don't fail, I guess that Bugsnag isn't active because of its own default configuration. Or is VCR recording it? It would be good if we could test Bugsnag integration when we add code to know more about certain bugs.

Reply:
Yes, actually I tried removing it from test but then realised we do have tests on specific bugsnag implementations ([for example](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/spec/models/stripe_account_spec.rb#L28-L33)). I didn't come across any instances where VCR records it, but I didn't look.

Like you say, we would need to test in dev sometimes, but that requires a manual config change anyway. I left a clue here so people can find it ;)